app/Ptr/DnsRecordService.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Packages\Rdns\App\Ptr;
+
+class DnsRecordService {
+  
+  /**
+   * @param string $domain
+   * 
+   * @return array 
+   */
+  public function getARecords ($domain) {
```

Comment:
all function's arguments and return type should be annotated with types (in this case string/array)